LISA/Executor.cpp

```diff
@@ -216,7 +216,12 @@ uint32_t Executor::Uninstall(const std::string& type,
         return ERROR_WRONG_PARAMS;
     }
 
-    if (! isAppInstalled(type, id, version)) {
+    if (version.empty() && !type.empty() && !id.empty() && uninstallType == "full") {
```

Comment:
What's the actual scenario here? How an app can fall into this case while uninstalling it?

Reply:
Added extra comments